src/Sulu/Bundle/ContentBundle/Controller/NodeController.php

```diff
@@ -776,6 +776,17 @@ public function postTriggerAction($uuid, Request $request)
                     $this->getDocumentManager()->removeDraft($data, $language);
                     $this->getDocumentManager()->flush();
                     break;
+                case 'restore':
```

Comment:
@chirimoya When thinking about it... Should we move that functionality to the `VersionController`, and deactivate the version controller as a whole?